src/main/java/ymfc/ui/Ui.java

```diff
@@ -200,4 +217,12 @@ public void printFind(ArrayList<Recipe> list, int listCount) {
         System.out.println("\tTotal: " + listCount + " recipes found!");
         System.out.println(LINE);
     }
+
+    public String getLine() {
```

Comment:
Should we just introduce `LINE` as a `public static final` variable?

Reply:
As of now, it's only being accessed in a single other class (test), so leaving it as private is fine?